.github/scripts/build.sh

```diff
@@ -3,15 +3,15 @@
 source `dirname ${BASH_SOURCE[0]}`/config-mingw.sh
 
 .github/scripts/binutils/build.sh
-if [ $BUILD_MINGW ]; then
+if [ $BUILD_MINGW = 1 ]; then
```

Comment:
I would expect it should be 
`if [ "$BUILD_MINGW" = 1 ];`

Reply:
Those two notations are, in its context, equivalent. Doing this change would demand also https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/104/files. Do you prefer to include those changes here?

Reply:
They are equivalent until the variable is defined, otherwise it will be an error
https://www.mycompiler.io/view/J3joke3OmC5